app/async/catch-link-click.js

```diff
@@ -32,7 +31,15 @@ exports.create = function (api) {
       var href = anchor.getAttribute('href')
       if (!href || href === '#') return
 
-      var url = Url.parse(href)
+      var url
+
+      try {
+        var url = new URL(href)
```

Comment:
should be just this I think
```js
url = new URL(href)
```

Reply:
I think I had to add this because it was throwing errors on invalid URIs. Unfortunately `new URL()` doesn't accept a callback to handle errors, so you *have* to `try {} catch {}` unless you're sure it's always a URI.